core/src/test/resources/sql/misc.iq

```diff
@@ -2467,10 +2477,62 @@ FROM "hr"."emps";
 
 !ok
 
-# [CALCITE-6566] JDBC adapter should generate PI function with parentheses in most dialects
-
 !use scott-mysql
 
+# [CALCITE-7270] Add support for a DIVIDE_0_NULL operator
+select 5 / 0 as a;
++---+
+| A |
++---+
+|   |
++---+
+(1 row)
+
+!ok
+
+# No NULL result produced for double division
+select CAST(5 AS DOUBLE) / 0 as a;
++----------+
+| A        |
++----------+
+| Infinity |
++----------+
+(1 row)
+
+!ok
+
+select 5 / 1 as a;
++---+
+| A |
++---+
+| 5 |
++---+
+(1 row)
+
+!ok
+
+# [CALCITE-7270] Add support for a DIVIDE_0_NULL operator
```

Comment:
nit: In some queries there is a comment with the JIRA ticket and summary and in others there isn't.